code/datums/components/material_container.dm

```diff
@@ -59,6 +59,9 @@
 	var/datum/callback/pc = precondition
```

Comment:
Actually, wait, there's a massive if up on L55 that runs before . is set to TRUE. Remove the message and just change the bit that says `I.flags_2 & HOLOGRAM_2` to `I.flags_2 & (HOLOGRAM_2 | NO_MAT_REDEMPTION_2)`

Reply:
Oh hell, how'd I miss that.